CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ This project employs a versioning scheme described in [RELEASE.md](RELEASE.md#ve
 
 - Fix not to send unassigned query parameters in Sabakan integration in [#725](https://github.com/cybozu-go/cke/pull/725)
 
+## [1.28.1]
+
+### Added
+
+- Take in updates of CKE 1.27.11. See [CKE 1.27.11](https://github.com/cybozu-go/cke/blob/v1.27.11/CHANGELOG.md#12711).
+- change CKE to proceed rebooting immediately after draining of node is completed [#707](https://github.com/cybozu-go/cke/pull/707)
```

Comment:
Sorry, I think L21,L22 should be placed on `## Changed` section. Could you fix this?

Reply:
I fixed it!

Reply:
Sorry, I cannot see the changes.
Could you check again please?